src/Uno.UWP/Globalization/ApplicationLanguages.cs

```diff
@@ -79,29 +110,6 @@ private static void ApplyLanguages()
 
 				Languages = languages.Distinct().ToArray();
 			}
-
-			var primaryLanguage = Languages.Count > 0 ? Languages[0] : null;
-
-			if (primaryLanguage is not null)
-			{
-				if (typeof(ApplicationLanguages).Log().IsEnabled(LogLevel.Debug))
-				{
-					typeof(ApplicationLanguages).Log().Debug($"Using {primaryLanguage} as primary language");
-				}
-
-				var primaryCulture = CreateCulture(primaryLanguage);
-
-				CultureInfo.CurrentCulture = primaryCulture;
-				CultureInfo.CurrentUICulture = primaryCulture;
```

Comment:
Concerned that we're no longer setting currentculture when primaryoverridelanguage changes - this would be a breaking change for uno apps that rely on this behaviour (perhaps??)

Reply:
Added a feature flag for that.